tests/app/test_setup.py

```diff
@@ -1,6 +1,9 @@
+from unittest.mock import MagicMock
```

Comment:
We should get this from pytest mock rather than unitttest, think we tried to move away from unittest outside of our integration tests 👍

Reply:
updated to use: `from mock import MaigcMock`